packages/react/src/wallet/ConnectWallet/screens/Metamask/MetamaskConnecting.tsx

```diff
@@ -1,12 +1,12 @@
-import { MetamaskIcon } from "../../icons/MetamaskIcon";
+import { MetamaskWallet } from "../../../wallets";
 import { ConnectingScreen } from "../ConnectingScreen";
 
 export const MetamaskConnecting: React.FC<{ onBack: () => void }> = (props) => {
   return (
     <ConnectingScreen
       onBack={props.onBack}
       walletName="Metamask"
-      WalletIcon={MetamaskIcon}
+      walletIconURL={MetamaskWallet.meta.iconURL}
       supportLink="https://support.metamask.io/hc/en-us/articles/4406430256539-User-Guide-Troubleshooting"
```

Comment:
I wonder if these should also live on the wallet meta

Reply:
it is coming from meta 🤔

Reply:
I think he's talking about the `supportLink`